presto-main/src/main/java/com/facebook/presto/type/BigintOperators.java

```diff
@@ -253,4 +255,21 @@ public static long hashCode(@SqlType(StandardTypes.BIGINT) long value)
     {
         return value;
     }
+
+    @ScalarOperator(IS_DISTINCT_FROM)
+    @SqlType(StandardTypes.BOOLEAN)
+    public static boolean isDistinctFrom(
+            @SqlType(StandardTypes.BIGINT) long left,
+            @IsNull boolean leftNull,
+            @SqlType(StandardTypes.BIGINT) long right,
+            @IsNull boolean rightNull)
+    {
+        if (leftNull != rightNull) {
+            return true;
+        }
+        if (leftNull && rightNull) {
```

Comment:
Write `if (leftNull) {` instead. Do the same everywhere else. You can add a comment before this line that says `// leftNull and rightNull are either both true or both false` if you think that's necessary. I'm okay with or without the comment.